tests/src/Kernel/UnitEntityTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\helfi_tpr\Kernel;
+
+use Drupal\helfi_tpr\Entity\Unit;
+
+/**
+ * Tests TPR Unit entities.
+ *
+ * @group helfi_tpr
+ */
+class UnitEntityTest extends MigrationTestBase {
```

Comment:
Can you add tests for other entities as well. Either rename this to something more generic (like EntityDeletionTest) or create separate `{entity_type}EntityTest classes`